### PR TITLE
docs(rbac): add group hierarchy support information

### PR DESCRIPTION
**Pull request by dzemanov**
## Description
Add information about leveraging group hierarchy.

## Fixes
https://issues.redhat.com/browse/RHIDP-3553

### CODE REVIEW

File: plugins/rbac-backend/docs/group-hierarchy.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    type: team
+    children: [group-b]
+  ```
+
+  ```CSV
+  g, group:default/group-a, role:default/test
+  p, role:default/test, catalog-entity, read, allow
+  ```
+
+- Sam wll inherit `role:default/test` via `group-a` <- `group-b` <- `group-c` <- `group-d`,
```
Comment (AndrienkoAleksandr) [inline plugins/rbac-backend/docs/group-hierarchy.md:144]:
```suggestion
- Sam will inherit `role:default/test` via `group-a` <- `group-b` <- `group-c` <- `group-d`,
```

File: plugins/rbac-backend/docs/group-hierarchy.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    parent: group-c
+  ---
+  apiVersion: backstage.io/v1alpha1
+  kind: Group
+  metadata:
+    name: group-d
+  spec:
+    type: team
+    children: [group-c]
+  ```
```
Comment (AndrienkoAleksandr) [inline plugins/rbac-backend/docs/group-hierarchy.md:173]: Maybe you need to provide csv file example here too.

Comment (dzemanov) [reply plugins/rbac-backend/docs/group-hierarchy.md:173]: Updated :)

**Review by AndrienkoAleksandr — COMMENTED** [review]
Looks good to me. @PatAKnight ?

**Review by PatAKnight — COMMENTED** [review]
Also looks good to me, just need to fix the `'yarn install', workspace is dirty!` error

**Review by PatAKnight — APPROVED** [review]
/lgtm

**Pull request merged**